Ravimotha patch 2 (#5601)
umbraco/UmbracoDocs: sustainability-best-practices/infrastructure/hosting.md

## Hosting - renewable and low-carbon energy

> All major cloud providers have committed to using renewable energy sources for their power needs. Some providers are going further and committing to more responsible utilization/recycling of water - thisis  a big deal, given the cooling requirements of data centers. Using a hosting provider that is powered by renewable will make a difference.

Reviewer: ```suggestion
All major cloud providers have committed to using renewable energy sources for their power needs. Some providers are going further and committing to more responsible utilization/recycling of water - this is a big deal, given the cooling requirements of data centers. Using a hosting provider that is powered by renewable will make a difference.
```

> The the Green Web Foundation has a [list of green hosts](https://www.thegreenwebfoundation.org/tools/directory/).

Reviewer: ```suggestion
The Green Web Foundation has a [published list of green hosts](https://www.thegreenwebfoundation.org/tools/directory/).
```

## SaaS/PaaS vs IaaS & On-prem

> Using **Software as a Service (SaaS) services** means you are offloading responsibility for a service to a third party so choose wisely. Use providers who can show they are using renewable energy and have committed to using reneewable energy and other sustainablity best practices. The best providers will balance the load on the underlying infrastructure so that overall utilization is high and they don’t have idle servers.

Reviewer: ```suggestion
Using **Software as a Service (SaaS) services** means you are offloading responsibility for a service to a third party so choose wisely. Use providers who can show they are using renewable energy and have committed to using renewable energy and other sustainability best practices. The best providers will balance the load on the underlying infrastructure so that overall utilization is high and they don’t have idle servers.
```
